frontend/projects/admin/src/app/event/event-dashboard/event-dashboard.component.html

```diff
@@ -3,6 +3,375 @@
+                  >{{ p.id }}<span data-ng-if="!$last">, </span></span
+                >. Please check that the provider support the selected currency or
+                the
+                <a
+                  data-ui-sref="configuration.organization({organizationId: event.organizationId})"
+                  >configuration</a
+                >.
+              </div> -->
+            </dd>
+            <dt class="col-sm-3">Max tickets</dt>
+            <dd class="col-sm-9">
+              {{ eventOrganizationInfo.event.availableSeats }}
+            </dd>
+            <dt
+              class="col-sm-3"
+              *ngIf="!eventOrganizationInfo.event.freeOfCharge"
+            >
+              VAT (%)
+            </dt>
+            <dd
+              class="col-sm-9"
+              *ngIf="!eventOrganizationInfo.event.freeOfCharge"
+            >
+              {{ eventOrganizationInfo.event.vatPercentage | currency : "" }}
+            </dd>
+            <dt
+              class="col-sm-3"
+              *ngIf="!eventOrganizationInfo.event.freeOfCharge"
+            >
+              Final Price
+            </dt>
+            <dd
+              class="col-sm-9"
+              *ngIf="!eventOrganizationInfo.event.freeOfCharge"
+            >
+              {{
+                eventOrganizationInfo.event.finalPrice
+                  | currency : eventOrganizationInfo.event.currency || ""
+              }}
+            </dd>
+          </dl>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12">
+          <hr />
+          <div class="d-flex flex-row-reverse">
+            <button class="btn btn-lg btn-primary">Edit</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="mt-5">
+    <h3>Categories</h3>
+    <h5 class="text-muted">
+      Here the categories that have been defined for this event.
+    </h5>
+  </div>
+  <button
+    type="button"
+    class="btn btn-success"
+    data-ng-disabled="!event.addCategoryEnabled"
+    data-ng-click="addCategory(event)"
+  >
+    <svg-icon key="add"></svg-icon> add category
+  </button>
+  <hr />
+  <div>
+    <label
+      class="btn btn-sm btn-default ng-pristine ng-untouched ng-valid ng-not-empty active"
+      data-ng-model="selection.active"
+      data-uib-btn-checkbox=""
+      ><i
+        class="fa fa-check-square-o"
+        data-ng-class="{'fa-check-square-o': selection.active, 'fa-square-o': !selection.active}"
+      ></i>
+      Active <span class="badge ng-binding">1</span></label
+    >
+    <label
+      class="btn btn-sm btn-default ng-pristine ng-untouched ng-valid ng-not-empty"
+      data-ng-model="selection.expired"
+      data-uib-btn-checkbox=""
+      ><i
+        class="fa fa-square-o"
+        data-ng-class="{'fa-check-square-o': selection.expired, 'fa-square-o': !selection.expired}"
+      ></i>
+      Expired <span class="badge ng-binding">0</span></label
```

Comment:
same as above. 
- please replace static number with actual count of expired categories
- ng-binding is not a valid class anymore, please remove it